index.html

```diff
@@ -0,0 +1,149 @@
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_2.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_3.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_4.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
```

Comment:
- [ ] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/192126381-c1ca8ae2-329e-4ea7-b9d9-43e007223e73.png)
### You design
![yourrrrrrrr](https://user-images.githubusercontent.com/81727988/192126388-cfcdecca-3280-4a00-afd5-e0ab7cdb1f3e.png)